python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -415,10 +499,22 @@ async def do_applyFocusOffset(self, id_data):
         """
         self.assert_enabled('applyFocusOffset')
 
-        self.model.set_offset("z", id_data.offset)
+        self.model.add_offset("z", id_data.offset)
+
+        # this isn't called when this routine is called when filter/disperser changes occur
+        # but will leave user_supplied here in case a routine wants to call it in the future
```

Comment:
You probably want to create a different method then. These `do_*` methods are not supposed to be called by users.